backend/src/handlers/reviews/main.go

```diff
@@ -41,9 +41,9 @@ func handler(ctx context.Context, req Request) (Response, error) {
 	case "GET":
 		_, ok := req.QueryStringParameters["sort"]
 		if ok {
-			return getReviews(ctx, req)
+			return getMultipleReviews(ctx, req)
```

Comment:
incredibly nitpicky but I prefer "(verb)(Noun)" for doing an action on one item and "(verb)(Noun)s" for an action on multiple. either way changing "getUserReview" to something else is good

Reply:
yeah I changed it to that at first but then I thought maybe they were too similar in name so I changed it again to this

Reply:
I get that. I think its fine tho, especially since all of the other functions already follow that. I would rather keep things consistent; we already have names like `MarshalUser / MarshalUsers` and `GetReview / GetReviews`. Im fine with renaming all of those to follow a convention like yours too, though I cant think of a very good one. I dont really like "(verb)Multiple(Noun)s" since that can be a lot to type

Reply:
nah I think it’s kinda jank too I’ll rename it once I get off the floor 😭